scania-digital-design-system/tegel
# fix(dropdown): z index

**Describe pull-request**  
Before dropdown later in the DOM had a naturally higher z-index than the ones before it. This caused issues when dropdowns were stacked after one another. 

**Solving issue**  
Fixes: [CDEP-2240](https://tegel.atlassian.net/browse/CDEP-2240)

**How to test**  
1. Go to Dropdown
2. Open the dropdown and make sure the one that is open is always displayed on top of the other dropdowns.

**Suggested test steps**
